Elevel/Application/Features/TopicCommands/GetTopicListQuery.cs

```diff
@@ -0,0 +1,76 @@
+﻿using AutoMapper;
+using Elevel.Application.Interfaces;
+using Elevel.Application.Pagination;
+using Elevel.Domain.Enums;
+using Elevel.Domain.Models;
+using MediatR;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Elevel.Application.Features.TopicCommands
+{
+    public class GetTopicListQuery
+    {
+        public class Request : PagedQueryBase, IRequest<Response>
+        {
+            public Guid? Id { get; set; }
+            public string TopicName { get; set; }
+            public DateTimeOffset CreationDate { get; set; }
+        }
+
+        public class Handler : IRequestHandler<Request, Response>
+        {
+            private readonly IApplicationDbContext _context;
+            private readonly IMapper _mapper;
+            
+            public Handler(IApplicationDbContext context, IMapper mapper)
+            {
+                _context = context;
+                _mapper = mapper;
+            }
+
+            public async Task<Response> Handle(Request request, CancellationToken cancellationToken)
+            {
+                var topicList = _context.Topics.AsNoTracking()
+                     .Include(x => x.Id)
```

Comment:
you should no include each property by name, I recommend to read for which purposes we have Include